cmd/symbols/internal/database/writer/writer.go

```diff
@@ -136,10 +136,10 @@ func (w *databaseWriter) writeFileIncrementally(ctx context.Context, args search
 		}
 
 		switch c.Status {
-		case gitdomain.AddedAMD, gitdomain.ModifiedAMD:
+		case gitdomain.StatusAdded, gitdomain.StatusModified:
 			addedOrModifiedPaths = append(addedOrModifiedPaths, c.Path)
 			addedModifiedOrDeletedPaths = append(addedModifiedOrDeletedPaths, c.Path)
-		case gitdomain.DeletedAMD:
+		case gitdomain.StatusDeleted:
 			addedModifiedOrDeletedPaths = append(addedModifiedOrDeletedPaths, c.Path)
 		}
```

Comment:
Same as other comments: 

Would it be more helpful for future readers to specifically enumerate the ones we deliberately ignore (e.x. gitdomain.StatusTypeChanged, ....

```suggestion
		case gitdomain.StatusTypeChanged:
			// Intentionally ignoring StatusTypeChanged
		}
```